Paper.tex

```diff
@@ -637,19 +686,26 @@ \subsubsection{Block Header Validity}
 \nonumber& & H_{\mathrm{l}} \geqslant 5000  \quad \wedge \\
```

Comment:
It might be relegated to a footnote, but in the check a couple of lines above that the gas limit may change only by less than ±1/1024 the parent gas limit is multiplied by the elasticity multiplier rho when H<sub>i</sub> = F<sub>London</sub>.

Reply:
I think the elasticity multiplier is irrelevant when validating the gas limit.

From the EIP:
```
		parent_gas_limit = self.parent(block).gas_limit
...
		# check if the block changed the gas limit too much
		assert block.gas_limit < parent_gas_limit + parent_gas_limit // 1024, 'invalid block: gas limit increased too much'
		assert block.gas_limit > parent_gas_limit - parent_gas_limit // 1024, 'invalid block: gas limit decreased too much'

```

Reply:
I'm talking about this bit:
```
		# on the fork block, don't account for the ELASTICITY_MULTIPLIER to avoid
		# unduly halving the gas target.
		if INITIAL_FORK_BLOCK_NUMBER == block.number:
			parent_gas_target = self.parent(block).gas_limit
			parent_gas_limit = self.parent(block).gas_limit * ELASTICITY_MULTIPLIER 
```

Reply:
ah, I missed that detail, thanks, I'll incorporate it

Reply:
I adjusted it like so:
![image](https://github.com/ethereum/yellowpaper/assets/13876978/387143fe-670f-4579-b91d-1c10b8db7291)

As well as the block header validity function:
![image](https://github.com/ethereum/yellowpaper/assets/13876978/4bfa68a2-eafa-47ff-9441-b828ca336cda)

Reply:
I didn't adjust the definition of the parent block gas target (tau), since this makes no difference to the base fee calculation, since it's a constant at the London fork block.